### PR TITLE
Reduce the installation size even more

### PR DESCRIPTION
**Pull request by SukkaW**
The PR follows #1394.

The PR reduces another 100 KiB from the installation size:

```bash
$ du -sh build
6.1M	build
```

### CODE REVIEW

**Comment by SukkaW** [comment]
CI failed, converting the PR to draft for now.

**Comment by SukkaW** [comment]
@raineorshine Now CI has passed, the PR is ready for review now.

File: src/lib/version-util.ts [hunk 1]
```diff
@@ -523,22 +519,22 @@ export function upgradeDependencyDeclaration(
...
-const revertLeadingV = curry((current: string, latest: string) => (v(current) ? v(current) + latest : latest))
+  /** Reverts a valid semver version to a pseudo version that is missing its patch components. NOOP If the original version was a valid semver version. */
+  const missingPatch = isMissingPatch(current)
+  result = missingPatch ? result.slice(0, result.length - '.0'.length) : result
 
-/** Reverts a valid semver version to a pseudo version. NOOP If the original version was a valid semver version. */
-const revertPseudoVersion = (current: string, latest: string) =>
-  flow(revertLeadingV(current), revertMissingMinorAndPatch(current), revertMissingPatch(current))(latest)
+  return result
+}
```
Comment (raineorshine) [inline src/lib/version-util.ts:537]: I'll accept this change, though a slightly preferable approach would have been to implement a simple flow function and preserve the existing structure of the code. (The currying was clearly unnecessary though.)

**Review by raineorshine — APPROVED** [review]
Thanks so much!

**Pull request merged**